### ISSUE TITLE
Improve Ruby and Bash support

### ISSUE THREAD

**Issue by KevinCorcoran** [Misophistful/borealis-cursive-theme#3]
Hi,

First of all, let me just say that Borealis is great!  I've been using it for a year or so at this point and find it just wonderful for Clojure and Java.  Thanks so much for creating and sharing it!

However, I work with a lot of Ruby as well, in the same projects as my Clojure/Java code, and Borealis looks really, really bad on Ruby code, to the point where I regularly switch away to a different color scheme.  Ditto for Bash scripts.

Any improvements that would make those languages show-up better in Borealis (or even just suggestions on how to tweak configuration settings in IntelliJ for a better experience without actually updating Borealis) would be greatly appreciated.  Thanks again!

**Comment by Misophistful** [comment]
Hi Kevin,

Unfortunately I only have the Community Edition of IDEA, which doesn't support Ruby (at least as far as I know). So it's difficult for me to understand exactly how the code looks bad, and how to fix it.

I'd suggest just playing around with the colours yourself in `Preferences -> Editor -> Colors & Fonts` until you get something that you're happy with.

I just downloaded the `BashSupport` plugin, and I see what you mean about Bash scripts looking ugly. I'll fix that in the next version I release.

**Comment by KevinCorcoran** [comment]
Hi James,

Thanks a lot for your response, and especially the intention to improve bash formatting in the next release!

Here is an image of some Ruby code in Borealis which should demonstrate what I mean about the colors making the code hard to read:

![ruby-borealis](https://cloud.githubusercontent.com/assets/2945462/14924464/94b8a3a4-0df7-11e6-8cf9-b0bb09bbe6b7.png)

I'll note a few specifics just to be clear:
- The dark blue method names are hard to read
- Note also `$stderr` on line 152 is basically invisible (as is the `catalog` variable on lines 171 and 175)
- The white background for string interpolation characters is quite jarring

If I get a chance to try tweaking the colors manually, I will let you know how it goes.  Thanks again!

**Comment by Misophistful** [comment]
Yes, I see how horrible it looks with Ruby files. I'll try to figure out a way to borrow a colleague's Ultimate edition, so I can fix it. Might take me a while though.

In the mean time, the latest version should look much better with Bash scripts. Please give it a try, and let me know if it's an improvement.

**Comment by KevinCorcoran** [comment]
Wow, thanks for the quick turnaround!  Bash definitely looks much better overall, however there's still one thing which I find pretty difficult to read - the dark blue words/commands.  Here's an example:

![bash-borealis](https://cloud.githubusercontent.com/assets/2945462/14928560/7b2dffd6-0e0d-11e6-9fde-fef0f003b5a6.png)

Other than that, it looks really good - I looked through a handful of scripts and didn't find anything else which looked like it needed improvement.

**Comment by Misophistful** [comment]
I've made another update, which hopefully fixes it. Please let me know.

**Comment by KevinCorcoran** [comment]
much better!
